libfreerdp/core/server.c

```diff
@@ -944,6 +966,13 @@ HANDLE WINAPI FreeRDP_WTSVirtualChannelOpen(HANDLE hServer, DWORD SessionId, LPS
 	hChannelHandle = (HANDLE) channel;
 
 	return hChannelHandle;
+
+error_queue:
+	Stream_Free(channel->receiveData, TRUE);
+error_receiveData:
+	free(channel);
```

Comment:
I'm sorry norbert I don't see the risk and why it's better to set channel to NULL. Anyway I have catched a case with the calloc of the channel beeing NULL.